static/app/views/alerts/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -138,6 +140,29 @@ class ActionsPanel extends PureComponent<Props> {
     onChange(triggerIndex, triggers, replaceAtArrayIndex(actions, index, newAction));
   }
 
+  conditionallyRenderHelpfulBanner(triggerIndex: number, index: number) {
+    const {triggers} = this.props;
+    const {actions} = triggers[triggerIndex];
+    const newAction = {...actions[index]};
+    if (newAction.type === 'slack') {
+      return (
+        <MarginlessAlert type="warning">
+          {tct(
+            'Having rate limiting problems? Enter a channel or user ID. Read more [rateLimiting].',
+            {
+              rateLimiting: (
+                <ExternalLink href="https://docs.sentry.io/product/integrations/notification-incidents/slack/#rate-limiting-error">
+                  {t('here')}
+                </ExternalLink>
+              ),
+            }
+          )}
+        </MarginlessAlert>
+      );
+    }
+    return null;
```

Comment:
```suggestion
    if (newAction.type !== 'slack') {
    	return null;
    }
    
    return (
      <MarginlessAlert type="warning">
        {tct(
          'Having rate limiting problems? Enter a channel or user ID. Read more [rateLimiting].',
          {
            rateLimiting: (
              <ExternalLink href="https://docs.sentry.io/product/integrations/notification-incidents/slack/#rate-limiting-error">
                {t('here')}
              </ExternalLink>
            ),
          }
        )}
      </MarginlessAlert>
    );
```

All about the [guard clause](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)

Reply:
Also you can write the external link `tct` like this 

```tsx
tct(
          'Having rate limiting problems? Enter a channel or user ID. Read more [rateLimiting:here].',
          {
            rateLimiting: (
              <ExternalLink href="https://docs.sentry.io/product/integrations/notification-incidents/slack/#rate-limiting-error" />
            ),
          }
        )
```